rules/windows/process_creation/proc_creation_win_net_start_service.yml

```diff
@@ -1,12 +1,12 @@
-title: Service Execution
+title: Services Started Via Net.EXE
 id: 2a072a96-a086-49fa-bcb5-15cc5a619093
 status: test
-description: Detects manual service execution (start) via system utilities.
+description: Detects usage of the "net.exe" command to start a service using the "start" flag
 references:
     - https://github.com/redcanaryco/atomic-red-team/blob/f339e7da7d05f6057fdfcdd3742bfcf365fee2a9/atomics/T1569.002/T1569.002.md
 author: Timur Zinniatullin, Daniil Yugoslavskiy, oscd.community
 date: 2019/10/21
-modified: 2021/11/27
+modified: 2023/02/01
 tags:
     - attack.execution
     - attack.t1569.002
```

Comment:
you could add the original file name in the selection (true for *holds up a fist* ALL THE RULES)

Reply:
I could but chose not to in this PR :)